src/Socket/messages-recv.ts

```
@@ -255,92 +255,94 @@ export const makeMessagesRecvSocket = (config: SocketConfig) => {
-						code: getBinaryNodeChildString(child, 'reaction'),
-						count: 1
+			switch (child.tag) {
+				case 'reaction':
+					const reactionUpdate = {
+						id: from,
+						server_id: child.attrs.message_id!,
+						reaction: {
+							code: getBinaryNodeChildString(child, 'reaction'),
+							count: 1
+						}
 					}
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)